pytripgui/leftmenu.py

```diff
@@ -529,53 +579,65 @@ def populate_tree(self):
         data.SetData("plans")
         self.plans_node = self.AppendItem(self.rootnode, "Plans", data=data)
 
-        # ctx = self.data.get_images().get_voxelplan()
-        for voi in self.data.get_vois():
+        ### ctx = self.data.get_images().get_voxelplan()
+        for voi in self.data.vdx.vois:
             data = wx.TreeItemData()
             data.SetData(voi)
-            item = self.AppendItem(self.structure_node, voi.get_name(), data=data)
+            item = self.AppendItem(self.structure_node, voi.name, data=data)
             img = self.image_list.Add(pytripgui.guihelper.get_empty_bitmap(self.icon_size[0],
                                                                            self.icon_size[1],
-                                                                           voi.get_color()))
-            voi.set_icon(img)
+                                                                           voi.color))
+            voi.icon = img
             self.SetItemImage(item, img, wx.TreeItemIcon_Normal)
-        for plan in self.data.get_plans():
+
+        for plan in self.data.plans:
             data = wx.TreeItemData()
             data.SetData(plan)
             p_id = self.AppendItem(self.plans_node, plan.name, data=data)
-            if len(plan.get_vois()):
-                item = self.get_or_create_child(p_id, "ROIs", plan.get_vois())
-                for voi in plan.get_vois():
+            if len(plan.vois):
```

Comment:
better to use simply `if plan.vois`
it will check if plan.vois is not None and is it is not emply